src/routes.js

```diff
@@ -13,8 +14,9 @@ const routes = (
 		<Route path="home" component={HomePage} />
 		<Route path="radio" component={RadioPage} />
 		<Route path="magazines" component={MagPage} />
+		<Route path="store" component={StorePage} />
```

Comment:
Maybe this is just me, but I think the Item Detail Component should have it's own route, maybe at `dialupstuff.com/store/item/102021` for example. We did something similar in the Rogue Agency site:

`<Route path="/artists/:id" component={IndividualArtistPage} />`

Maybe we could use `<Route path="/store/item/:id" component={StoreItemComp} />`